dovecot/plan.sh

```diff
@@ -0,0 +1,49 @@
+pkg_name=dovecot
+pkg_origin=core
+pkg_version=2.3.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="Secure IMAP server"
+pkg_upstream_url="https://dovecot.org"
+pkg_license=("LGPL-2.1" "MIT")
```

Comment:
SPDX has updated their [license strings](https://spdx.org/licenses/), and this should be `LGPL-2.1-only` now.